xcode/Subconscious/Shared/Components/FirstRun/FirstRunDoneView.swift

```diff
@@ -12,7 +12,11 @@ struct FirstRunDoneView: View {
     @Environment(\.colorScheme) var colorScheme
     
     private var status: ResourceStatus {
-        app.state.gatewayProvisioningStatus
+        if app.state.lastGatewaySyncStatus == .succeeded {
+            return app.state.lastGatewaySyncStatus
+        }
```

Comment:
Edge case: if you end up on this screen after relaunching the app and have sync'd with the gateway then we should say you're connected.